spec/expand/dollar_sign/dollar_operator.spec

```diff
@@ -0,0 +1,8 @@
+# outside of a cluster definition the dollar sign is not a metacharacter
+$foo
+$foo
```

Comment:
is there a good use case for this? Does it make more sense for this to be a parse error (or at least no results)?